projects/1.18/assets/tetra-pak/tetrapak/lang/zh_cn.json

```
@@ -108,81 +108,81 @@
+  "tetra.improvement.tetrapak/reaping.description": "此工具附有的黑魔法使其能将击杀的亡灵生物转换为魂之碎片",
   "tetra.improvement.tetrapak/cleaving.name": "开劈之斩",
-  "tetra.improvement.tetrapak/cleaving.description": "此工具被黑魔法磨得比剃须刀还锋利，从而使其击杀的生物有概率掉落它们的头颅",
+  "tetra.improvement.tetrapak/cleaving.description": "此工具被黑魔法磨得比剃须刀还锋利，从而偶尔使其击杀的生物掉落它们的头颅",
   "tetra.improvement.tetrapak/bubbling.name": "泡泡",
   "tetra.improvement.tetrapak/bubbling.description": "此物品产生的所有粒子效果都被换成了泡泡。真有趣！",
   "tetrapak.stats.reaping": "死神收割",
-  "tetrapak.stats.reaping.tooltip": "此物品会使亡灵生物掉落灵魂碎片",
+  "tetrapak.stats.reaping.tooltip": "此物品会使亡灵生物掉落魂之碎片",
   "tetrapak.stats.cleaving": "开劈之斩",
-  "tetrapak.stats.cleaving.tooltip": "此物品会使玩家和其他生物有概率掉落它们的头颅",
+  "tetrapak.stats.cleaving.tooltip": "此物品会使玩家和其他生物偶尔掉落它们的头颅",
   "tetrapak.stats.voiding": "虚空",
-  "tetrapak.stats.voiding.tooltip": "此物品会清除其击杀生物的和其采集方块的掉落物",
+  "tetrapak.stats.voiding.tooltip": "此物品会清除其击杀生物的和其采集方块的掉落物。",
   "tetrapak.stats.rainbow": "彩虹",
-  "tetrapak.stats.rainbow.tooltip": "此物品采集§e%s§r方块的速度§e+%s%%§r",
+  "tetrapak.stats.rainbow.tooltip": "此物品在采集§e%s§r方块的速度加快§e%s%%§r",
   "tetrapak.stats.floating": "飘浮",
   "tetrapak.stats.floating.tooltip": "此物品能反制重力，使得此物品收集到的掉落物和其自身飘浮",
   "tetrapak.stats.deforesting": "毁林",
-  "tetrapak.stats.deforesting.tooltip": "此物品会在砍伐一棵树最下方方块后砍倒整棵树",
+  "tetrapak.stats.deforesting.tooltip": "此物品会在破坏一棵树最下方方块后砍倒整棵树",
   "tetrapak.stats.collapsing": "坍塌",
-  "tetrapak.stats.collapsing.tooltip": "此物品会在采集方块后，同时采集上方所有可被锹挖掘的方块（同种方块或可被重力影响的方块）",
+  "tetrapak.stats.collapsing.tooltip": "此物品会在挖下方块后，同时挖下上方所有可被锹采集的方块（同种方块或可被重力影响的方块）",
   "tetrapak.stats.unearthing": "出土",
-  "tetrapak.stats.unearthing.tooltip": "此物品会在开采一个矿石后开采同矿脉中最多\u00a7e%s\u00a7r个方块",
+  "tetrapak.stats.unearthing.tooltip": "此物品会在挖下一个矿石后挖掘同矿脉中最多\u00a7e%s\u00a7r个方块",
   "tetrapak.stats.nullifying": "反重力",
-  "tetrapak.stats.nullifying.tooltip": "穿着此腰带时受到的重力\u00a7e-%s\u00a7r",
+  "tetrapak.stats.nullifying.tooltip": "穿着此腰带时受到的重力会减弱\u00a7e%s\u00a7r",
   "tetrapak.stats.moonstrike": "月击",
-  "tetrapak.stats.moonstrike.tooltip": "此物品在满月时造成的伤害\u00a7e+%s%%\u00a7r，并使破坏方块的速度\u00a7e+%s%%\u00a7r。其余月相时增益量会减少",
+  "tetrapak.stats.moonstrike.tooltip": "此物品会在满月时造成\u00a7e%s%%\u00a7r的额外伤害，并使破坏方块的速度增加\u00a7e%s%%\u00a7r。其余月相时增益量会减少",
   "tetrapak.stats.regrowth": "再生",
-  "tetrapak.stats.regrowth.tooltip": "此物品每\u00a7e%s\u00a7r秒回复其总耐久度的\u00a7e1%%\u00a7r\n\n\u00a77总共需要大约\u00a7e%s\u00a77分钟以完全修复",
+  "tetrapak.stats.regrowth.tooltip": "此物品每\u00a7e%s\u00a7r秒回复其总耐久的\u00a7e1%%\u00a7r\n\n\u00a77总共需要大约\u00a7e%s\u00a77分钟以完全修复",
   "tetrapak.stats.scorching": "灼烈",
   "tetrapak.stats.scorching.tooltip": "此物品会烧灼掉落物，就像把它们放进熔炉中烧炼一样。对不可烧炼的物品无效",
   "tetrapak.stats.moonsight": "月视",
   "tetrapak.stats.moonsight.tooltip": "穿着此腰带能让你在月光下看得更清楚",
   "tetrapak.stats.expboost": "经验加成",
-  "tetrapak.stats.expboost.tooltip": "采集方块和击杀生物产生的经验\u00a7e+%s\u00a7r",
+  "tetrapak.stats.expboost.tooltip": "挖下的方块和击杀的生物会产生\u00a7e%s\u00a7的额外经验",
```

这里貌似原翻译还OK？严格按原文的话感觉有点怪
```suggestion
  "tetrapak.stats.expboost.tooltip": "使用此工具挖掘方块和击杀生物时，会产生\u00a7e%s\u00a7的额外经验",
```